docs/source/embedding.md

```diff
@@ -71,59 +71,116 @@ the `application/vnd.jupyter.widget-state+json` format specified in the
+      integrity="sha256-Ae2Vz/4ePdIu6ZyI/5ZGsYnb+m0JlOmKPjt6XZ9JJkA=" 
+      crossorigin="anonymous">
+    </script>
+    <script 
+      src="https://unpkg.com/@jupyter-widgets/html-manager@*/dist/embed-amd.js" 
+      crossorigin="anonymous">
+    </script>
+
+    <script type="application/vnd.jupyter.widget-state+json">
+      {manager_state}
+    </script>
+  </head>
+
+  <body>
+    <div id="first-slider-widget">
+      <script type="application/vnd.jupyter.widget-view+json">
+        {widget_views[0]}
+      </script>
+    </div>
+    <hrule />
+    <div id="second-slider-widget">
+      <script type="application/vnd.jupyter.widget-view+json">
+        {widget_views[1]}
+      </script>
+    </div>
+  </body>
+</html>
+"""
+
+manager_state = json.dumps(data['manager_state'])
+widget_views = [json.dumps(view) for view in data['widget_views']]
+rendered_template = html_template.format(manager_state=manager_state, widget_views=widget_views)
+with open('export.html', 'w') as fp:
+    fp.write(rendered_template)
+```
 
-The `embed_snippet` function will return an embeddable HTML snippet similar to the
-Notebook interface detailed above, while `embed_minimal_html` saves an HTML file
-with the snippet. The `embed_data` function will return the widget state JSON as
-well as the view specs of the given views.
+The web page needs to load RequireJS and the Jupyter widgets HTML manager.
+You then need to include the manager state in a `<script>` tag of type
+`application/vnd.jupyter.widget-state+json`, which can go in the head of the
+document. For each widget view, place a `<script>` tag of type
+`application/vnd.juptyer.widget-view.json` in the DOM element that should
+contain the view. The widget manager will replace each `<script>` tag with
+the DOM tree corresponding to the widget.
 
-In all functions, the state of all widgets known to the widget manager is
+In all embedding functions, the state of all widgets known to the widget manager is
 included by default. You can alternatively pass a reduced state to use instead.
 This can be particularly relevant if you have many independent widgets with a
 large state, but only want to include the relevant ones in your export. To
 include only the state of the views and their dependencies, use the function
 `dependency_state`:
 
 ```py
-s1, s2 = IntSlider(max=200, value=100), IntSlider(value=40)
+from ipywidgets.embed import embed_snippet, dependency_state
+
+s1 = IntSlider(max=200, value=100)
+s2 = IntSlider(value=40)
 print(embed_snippet(
     views=[s1, s2],
     state=dependency_state([s1, s2]),
     ))
 ```
 
-In the `embed_snippet` and `embed_minimal_html` examples above, the `requirejs=True` gives the RequireJS embedder. To get the standard embedder, omit this option or give `requirejs=False`.
+In the `embed_snippet` and `embed_minimal_html` examples above, the
+`requirejs=True` gives the RequireJS embedder. To get the standard embedder,
+omit this option or give `requirejs=False`.
```

Comment:
This is a little confusing, since `requirejs=True` is not explicitly stated (it is left as implicit via default values). Maybe just add the word 'default' here: "..., the default `requirejs=True` gives... 

Further, I know this is not from this PR, but I also think it is a little strange to call the non-default embedder "the standard embedder".

Also, if there is somewhere the difference between these embedders are explained, it would be nice with a link.

Reply:
Agreed. I actually think we should remove this paragraph completely. In this section of the documentation, we either describe using `embed_minimal_html`, which really should just work out of the box, so you need requireJS anyway, or embed_data, which is agnostic to how you import requireJS.

Unless I'm missing something, being able to pass `requireJS = False` is only useful for `embed_snippet`, which isn't really described here any more.